src/Surfnet/StepupGateway/GatewayBundle/Service/SecondFactorLocaleProvider.php

```diff
@@ -1,4 +1,19 @@
 <?php
+/**
+ * Copyright 2020 SURFnet bv
```

Comment:
Nitpick: when adding new copyright headers, please use the correct `B.V.` version.